eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -192,8 +192,17 @@ function UpdateDocsMsMetadataForPackage($packageInfoJsonLocation) {
   }
   $packageMetadataName = Split-Path $packageInfoJsonLocation -Leaf
   $packageInfoLocation = Join-Path $DocRepoLocation "metadata/$metadataMoniker"
-  $packageInfoJson = ConvertTo-Json $packageInfo
-  New-Item -ItemType Directory -Path $packageInfoLocation -Force
+  $docsMetadata = $packageInfo
+  if (Test-Path $packageInfoLocation) {
+    $docsMetadata = Get-Content "$packageInfoLocation/$packageMetadataName" -Raw | ConvertFrom-Json
+    $docsMetadata.PSObject.Properties | ForEach-Object {
+      $packageInfo | Add-Member -MemberType $_.MemberType -Name $_.Name -Value $_.Value -Force
```

Comment:
Does Add-Member work if we are updating a property?

Reply:
It looks like we are overriding the values with the docsMetadata from the docs repo I would have expected the other direction.

Reply:
Now logic goes: (Docs metadata means the json file from docs repo, package json means the json file from release pipeline.)
1. For each docs metadata property:
  a. If package json has the property, then do nothing .
  b. If package json doesn't have the property, then update to package json.

2. Then write package json back to docs repo.